tests/Handler/StreamHandlerTest.php

```diff
@@ -286,7 +286,7 @@ public function testAddsProxyButHonorsNoProxy()
             'no'   => ['*']
         ]]);
         $opts = stream_context_get_options($res->getBody()->detach());
-        $this->assertTrue(empty($opts['http']['proxy']));
+        $this->assertArrayNotHasKey($opts['http']['proxy']);
```

Comment:
Looks like this assertion does not handle undefined indexes as well. So, let's keep as it is? :)